<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QWidget" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>GhostFight</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/resources/ghost.png</normaloff>:/images/resources/ghost.png</iconset>
  </property>
  <widget class="QPushButton" name="GameSettingButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-image: url(:/images/resources/blankBack.png);
color: rgb(255, 0, 4);
font: 16pt &quot;Segoe Print&quot;;
}
QPushButton::hover{
	color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
}</string>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="RedScoreTextLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: italic 24pt &quot;Monotype Corsiva&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RedScoreLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: italic 24pt &quot;Monotype Corsiva&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BlueScoreTextLabel">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>0</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: italic 24pt &quot;Monotype Corsiva&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>Score:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BlueScoreLabel">
   <property name="geometry">
    <rect>
     <x>964</x>
     <y>0</y>
     <width>60</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: italic 24pt &quot;Monotype Corsiva&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="RedLifeBar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	color: rgb(242, 255, 124);
	background-color: rgb(0, 0, 0);
	font: 700 9pt &quot;Microsoft YaHei UI&quot;;
}
QProgressBar::chunk{
	background-color: rgb(255, 93, 61);
}
</string>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v/%m</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="BlueLifeBar">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	color: rgb(242, 255, 124);
	background-color: rgb(0, 0, 0);
	font: 700 9pt &quot;Microsoft YaHei UI&quot;;
}
QProgressBar::chunk{
	background-color: rgb(80, 132, 255);
}
</string>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="format">
    <string>%v/%m</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeCounter">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>150</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-image: url(:/images/resources/blankBack.png);
	color: rgb(0, 0, 127);
font: 16pt &quot;Segoe Print&quot;;
}</string>
   </property>
   <property name="text">
    <string>01:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>GameSettingButton</zorder>
  <zorder>RedScoreTextLabel</zorder>
  <zorder>RedScoreLabel</zorder>
  <zorder>BlueScoreTextLabel</zorder>
  <zorder>BlueScoreLabel</zorder>
  <zorder>RedLifeBar</zorder>
  <zorder>BlueLifeBar</zorder>
  <zorder>TimeCounter</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
